docs/competition/team-support/tinker-time.md

```diff
@@ -0,0 +1,13 @@
+# Tinker Time
+
+Tinker Time is time allocated in the schedule to allow teams to test their robot in the arena outside of a match.
+
+During Tinker Time up to 3 members of a team may enter the arena.
+
+To book Tinker Time teams must find the Tinker Time Coordinator, stationed by a whiteboard near the staging area, who will allocate teams slots based on what's available.
+
+Each 'slot' will last 10 minutes, 4 teams can be in the arena at once (one in each corner), robots must enter the arena through staging. However during Tinker Time the second door will be opened in scafolding and non robot carrying comeptitors may enter and exit through here.
```

Comment:
Please could we note explicitly that the comment about the second door is specific to 2019?

Perhaps something like:
```suggestion
Each 'slot' will last 10 minutes, 4 teams can be in the arena at once (one in each corner), robots must enter the arena through staging. For SR2019, where we have a scaffold-based arena, during Tinker Time the second door will be opened and non-robot-carrying competitors may enter and exit through here.
```